user-docs/modules/ROOT/pages/fdo-terminology.adoc

```diff
@@ -0,0 +1,43 @@
+= FIDO device onboarding terminology
+
+Learn more about the FIDO device onboarding terminology.
+
+.FDO terminology
+
+|===
+|Commands | Description
```

Comment:
Is it a command? It seems more a term given the above section header

Reply:
Honestly, this document was copied from the downstream documentation and I haven't reviewed it :(

Reply:
Changed